eng/Version.Details.xml

```diff
@@ -57,241 +57,241 @@
       <Uri>https://github.com/aspnet/EntityFrameworkCore</Uri>
       <Sha>e37a6be067ab62da7fe9b33b93faaa90ed829237</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.AspNetCore.Analyzer.Testing" Version="5.0.0-alpha1.19602.5" CoherentParentDependency="Microsoft.EntityFrameworkCore">
+    <Dependency Name="Microsoft.AspNetCore.Analyzer.Testing" Version="5.0.0-alpha1.19603.1" CoherentParentDependency="Microsoft.AspNetCore.Tooling">
```

Comment:
This is completely incorrect as I understand Maestro++ and `darc`. @mmitche please confirm.

 `CoherentParentDependency` names a _package_, not a repo. Since there's nothing named Microsoft.AspNetCore.Tooling, you're likely to have cause problems for Maestro++ and certainly aren't keeping things coherent. I suggest `CoherentParentDependency="Microsoft.CodeAnalysis.Razor"` or `CoherentParentDependency="Microsoft.NET.Sdk.Razor"`

Reply:
@pranavkm once @mmitche confirms, please open another PR.

Reply:
@dougbu you're right, I looked over this too quickly. It just so happens that there's a package named `Microsoft.EntityFrameworkCore`, which we used for the CoherentParentDependency. There is not a package named `Microsoft.AspNetCore.Tooling`